src/commands/githubActions/bumpUpstream/index.ts

```diff
@@ -153,6 +157,24 @@ Compose - ${JSON.stringify(compose, null, 2)}
 
   const versionsToUpdate = Array.from(versionsToUpdateMap.values());
   manifest.upstreamVersion = getUpstreamVersionTag(versionsToUpdate);
+
+  const ethProvider = await tryEthProviders({
+    providers: ["dappnode", "infura"]
```

Comment:
Arent these providers somewhere in params already?

Reply:
In order places of the code the provider is being passed as a string like here